src/Tribe/Importer/File_Importer_Venues.php

```diff
@@ -36,30 +35,162 @@ protected function create_post( array $record ) {
+			'Description' => 'venue_description',
+			'Phone'       => 'venue_phone',
+			'Province'    => 'venue_state',
+			'State'       => 'venue_state',
+			'URL'         => 'venue_url',
+			'Zip'         => 'venue_zip',
+		];
+
+		foreach ( $columns as $name => $key ) {
+			// Don't overwrite unset columns.
+			if ( ! $this->has_value_by_key( $record, $key ) ) {
+				continue;
+			}
+
+			// Address is a concatenation of two possible values.
+			if ( 'venue_address' === $key ) {
+				$address_1 = trim( $this->get_value_by_key( $record, 'venue_address' ) );
+				$address_2 = trim( $this->get_value_by_key( $record, 'venue_address2' ) );
+				$venue[ $name ] = ( empty( $address_2 ) ) ? $address_1 : $address_1 . ' ' . $address_2;
+
+				continue;
+			}
+
+			$venue[ $name ] = $this->get_value_by_key( $record, $key );
+		}
+
+		// Handle the manual stuff.
+		$venue['FeaturedImage'] = $this->get_featured_image( $venue_id, $record );
+
+		$show_map_setting       = tribe( 'events-aggregator.settings' )->default_map( 'csv' );
+		$venue['ShowMap']       = $venue_id ? get_post_meta( $venue_id, '_VenueShowMap', true ) : $show_map_setting;
+		$venue['ShowMapLink']   = $venue_id ? get_post_meta( $venue_id, '_VenueShowMapLink', true ) : $show_map_setting;
+
+		/**
+		 * Allows triggering using the default values set in the admin for imported venues.
+		 *
+		 * @since TBD
+		 * @param int   $venue_id The ID of the venue we're currently importing.
+		 * @param array $record The event record from the import.
+		 * @param array $venue The array of venue data we're modifying.
+		 */
+		$set_defaults = apply_filters(
+			'tribe_events_importer_set_default_venue_import_values',
+			false,
+			$venue_id,
+			$record,
+			$venue
 		);
 
-		if ( empty( $venue['Country'] ) ) {
-			$venue['Country'] = 'United States';
+		if ( $set_defaults ) {
+			$venue = $this->set_defaults( $venue, $record );
 		}
 
-		$venue = apply_filters( 'tribe_events_importer_venue_array', $venue, $record, $venue_id, $this );
+		/**
+		 * Allows filtering of values before import.
+		 *
+		 * @since 4.2
+		 *
+		 * @param array $venue The array of venue data we're modifying.
+		 * @param array $record The event record from the import.
+		 * @param int   $venue_id The ID of the venue we're currently importing.
+		 */
+		$venue = apply_filters(
+			'tribe_events_importer_venue_array',
+			$venue,
+			$record,
+			$venue_id,
+			$this
+		);
+
+		return $venue;
+	}
+
+	/**
+	 * Set default venue values.
+	 *
+	 * @since TBD
+	 *
+	 * @param array $venue  The array of venue data we're modifying.
+	 * @param array $record The event record from the import.
+	 * @return array        The modified venue data.
+	 */
+	public function set_defaults( $venue, $record ) {
+		$columns        = [
+			'Address'     => 'address',
+			'City'        => 'city',
+			'Country'     => 'country',
+			'Phone'       => 'phone',
+			'Province'    => 'state',
+			'State'       => 'state',
+			'URL'         => 'url',
```

Comment:
:warning: Array double arrow not aligned correctly; expected 6 space(s) between "'URL'" and double arrow, but found 9.

<sub>hash: dad7653cf94526869b8f1932b9928b15</sub>